ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/config/SandboxConfig.scala

```diff
@@ -46,13 +46,16 @@ final case class SandboxConfig(
     seeding: Option[Seeding],
     metricsReporter: Option[MetricsReporter],
     metricsReportingInterval: Duration,
+    eventsPageSize: Int,
 )
 
 object SandboxConfig {
   val DefaultPort: Port = Port(6865)
 
   val DefaultMaxInboundMessageSize: Int = 4 * 1024 * 1024
 
+  val DefaultEventsPageSize: Int = 10000
```

Comment:
`10000` seems quite large imho.

Reply:
Sure, what would you think would be a more suitable default? 1000?

Reply:
Well, I set it to 1000 in https://github.com/digital-asset/daml/pull/5305/commits/c5099b7aadf381bd236b3cebf806dc55e7aeceb2, let me know if that's ok.